tests/transformers/test_scale.py

```diff
@@ -0,0 +1,134 @@
+from pandas import DataFrame
+from sklearn.preprocessing import MinMaxScaler as SklearnMinMaxScaler
+from sklearn.preprocessing import Normalizer as SklearnNormalizer
+from sklearn.preprocessing import StandardScaler as SklearnStandardScaler
+
+from pyreal.transformers.scale import MinMaxScaler, Normalizer, StandardScaler
+
+# MinMaxScaler tests
+data = [[2, 1, 3, 10], [4, 3, 4, 0], [6, 7, 2, 2]]
+columns = ["A", "B", "C", "D"]
+
+# test whether pyreal and sklearn scalers behave the same
+
+
+def test_fit_minmax():
+    skMinMax = SklearnMinMaxScaler()
```

Comment:
Please be sure to follow the PEP8 style guide for naming, meaning `camel_case` names (`sk_min_max`).